### PR TITLE
cherry-pick logging updates into release-2.1

### PR DESCRIPTION
**Pull request by STRRL**
### What problem does this PR solve?

it is a part of https://github.com/chaos-mesh/chaos-mesh/issues/2969

### What's changed and how it works?

- cherry pick #2808 into release-2.1

### Related changes

- [ ] Need to update `chaos-mesh/website`
- [ ] Need to update `Dashboard UI`
- Need to **cheery-pick to release branches**
  - [ ] release-2.1
  - [ ] release-2.0

### Checklist

Tests

- [ ] Unit test
- [x] E2E test
- [ ] No code
- [x] Manual test (add steps below)

Side effects

- [ ] Breaking backward compatibility

### Release note 

```text
Please add a release note.

You can safely ignore this section if you don't think this PR needs a release note.
```

### DCO

If you find the DCO check fails, please run commands like below (Depends on the actual situations. For example, if the failed commit isn't the most recent) to fix it:

```shell
git commit --amend --signoff
git push --force
```

### CODE REVIEW

File: cmd/chaos-controller-manager/main.go [hunk 1]
```diff
@@ -17,6 +17,7 @@ package main
 
 import (
 	"flag"
+	stdlog "log"
 	"net/http"
 	_ "net/http/pprof"
```
[the remark this answers is not in the document]
Comment (STRRL) [reply cmd/chaos-controller-manager/main.go:22]: @sonatype-lift ignore

**Comment by STRRL** [comment]
PTAL

/cc @YangKeao @iguoyr @Andrewmatilde

**Review by YangKeao — APPROVED** [review]
LGTM

**Review by iguoyr — APPROVED** [review]
LGTM

**Pull request merged**